core/controllers/creator_dashboard.py

```diff
@@ -50,6 +50,15 @@
 QUESTION_ID_KEY = 'questionId'
 
 
+class OldNotificationsDashboardUrlHandler(base.BaseHandler):
+    """Redirects the old notifications dashboard URL to the new one."""
+
+    @acl_decorators.open_access
+    def get(self):
+        """Handles GET requests."""
+        self.redirect(feconf.NOTIFICATIONS_DASHBOARD_URL)
```

Comment:
Hi @iamprayush, here and elsewhere, can you explictly pass `permanent=True` so that the response code is set as 301 (and not 302). Here's the [link](https://webapp2.readthedocs.io/en/latest/api/webapp2.html#webapp2.redirect) to the webapp2 redirect method documentation. AFAIK, 301 is the response code used for permanent redirection and 302 is for temporary redirection (eg. when site is down for maintenance). So 301 is probably what we need here, right?

Reply:
Yup, done.

Reply:
Thanks! Can you also update the PR description with the new changes?
ie. 302 -> 301 and /teach#playbook change

Reply:
Done.